src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerAssemblyLoader.Desktop.cs

```diff
@@ -31,6 +31,28 @@ internal AnalyzerAssemblyLoader()
         {
         }
 
+        public bool IsHostAssembly(Assembly assembly)
+        {
+            // When an assembly is loaded from the GAC then the load result would be the same if 
+            // this ran on command line compiler. So there is no consistency issue here, this 
+            // is just runtime rules expressing themselves.
```

Comment:
It's a bit weird that the API is named `IsHostAssembly` but the comments are talking about "consistency" - they made sense when inside the consistency checker but here are bit out of context. But I guess it's fine since it's an internal API (and has just one caller - the consistency checker).